framework/components/AFeatureDiscovery/components/ATargetHighlight.js

```diff
@@ -0,0 +1,38 @@
+import React, {useEffect, useState} from "react";
+
+const ATargetHighlight = ({targetRef, padding = 10}) => {
+  const calcStyles = (padding, targetRef) => {
+    const elementRect = targetRef.current.getBoundingClientRect();
```

Comment:
can use `getRoundedBoundedClientRect` from `framework/utils/helpers` to clean this up a little